package.json

```diff
@@ -31,5 +34,8 @@
         "prettier": "^2.2.1",
         "typescript": "4.9.4"
     },
-    "packageManager": "yarn@3.1.0"
+    "packageManager": "yarn@3.1.0",
+    "resolutions": {
+        "prettier": "2.2.1"
```

Comment:
Pinning `prettier` to `2.2.1` since the latest version enforces some new rules that necessitate a complete reformatting of our codebase for successful linting.

To do this, we've added resolutions in our root package.json. This ensures we're always using the right `prettier` version, even if yarn.lock is removed. It's an alternative to [yarn set resolution](https://yarnpkg.com/cli/set/resolution) and is more reliable because it stays in place even if `yarn.lock` is regenerated.

But this is just a temporary solution. We'll eventually need to update `prettier` and reformat our code to meet the new rules in another PR.

Reply:
Okay, that works, we can just remove this in the version updates PR :+1: